src/wp-admin/comment.php

```diff
@@ -62,7 +62,8 @@
 
 		$comment_id = absint( $_GET['c'] );
 
-		if ( ! $comment = get_comment( $comment_id ) ) {
+		$comment = get_comment( $comment_id );
+		if ( ! $comment ) {
```

Comment:
Shouldn't it be like this

```
$comment_id = absint( $_GET['c'] );
$comment      = get_comment( $comment_id );

if ( ! $comment ) {
```

Declaring the variables first with equal sign indentation? Then whitespace to the next line? I might be confusing this with Woocommerce/Classic-commerce CS style.

Reply:
You are right about the alignment, the rule is `WordPress.WhiteSpace.PrecisionAlignment` but it is currently returning a 'Warning` rather than and `Error` to these remain, as yet, unaddressed.

Reply:
I will skip reviewing this particular rule as it is recurring in the codebase.